### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by liu1700**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,13 @@ $ ./arion -src <your.any.pb.go> -u
 Then by default Arion will generate a temporary folder containing source files and compile those files into an executable binary called *Postgal*. You can use `-o` to specify other path/name for this executable file. Also you can use `-c` to clear temp folder after *Postgal* is generated.
 
 *NOTE*
+
+If you get compiling error after running
+```bash
+./arion -src <your.any.pb.go> -u
+```
+try to create a temp folder in your $GOPATH/src and copy your.any.pb.go file to that folder and try again
```
Comment (LitoMore) [inline README.md]: I think this sentence below would be better

```
Please create a temp folder in your $GOPATH/src and copy `your.any.pb.go` file to the temp folder then try again
```

**Review by LitoMore — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (liu1700) [reply README.md]: Yeah, you are right. Updated!

**Review by LitoMore — APPROVED** [review]

**Comment by straightdave** [comment]
The solution mentioned is not accurate. Technically you can run the `arion` command everywhere to generate the postgals. The root cause of compiling error varies a lot. You can try to file an *issue* and describe how you met the problem and provide logs and error messages.

In a lot of cases I've met about compiling error they are mainly caused by:
* package conflict (you call `arion` to compile in your project folder with _vendor_ directory)
* underlying packages are not up-to-date

In the first case, you just call `arion` anywhere else, not in the projects with _vendor_ directories. For the second, you can use `-u` when running `arion` to force-update underlying packages.

**Comment by straightdave** [comment]
But thanks all the same.

**Comment by liu1700** [comment]
It might be package conflict in my case.

**Pull request closed without merging**